groups/rds/vault.tf

```diff
@@ -0,0 +1 @@
+vault-providers/userpass
```

Comment:
This won't work. You need to copy the `vault-providers` folder as it is [here](https://github.com/companieshouse/kafka-stack/tree/main/groups/grafana/vault-providers). You'll then need to delete the `vault.tf` file and link it to the `userpass` file. I.e. cd to the `groups/rds` folder and then run this `ln -s vault-providers/userpass vault.tf`